WomenWhoCode/WWCodeGreenville
# Make an auto labeler action

# Pull Request

## Description
I am adding a github action that automatically labels pull requests.

## Motivation
solves issue #132 

## Testing Procedures
make changes in any file/add a new file in any folder and create a pull request for it. the labeler will run a check and add necessary labels. 

## Checklist
- [yes ] Code adheres to project coding standards.
- [yes ] Tests cover the added or modified code.
- [yes ] Documentation has been updated, if necessary.
- [yes ] All new and existing tests pass successfully.
- [yes ] The pull request is up to date with the latest changes from the main branch.

## Reviewer Guidelines
i labeled "assets/readme" as documentation cause it affects the readme. Pls let me know your thoughts on it. 

## Screenshots or Visuals
![image](https://github.com/WomenWhoCode/WWCodeGreenville/assets/118920744/eab0b15f-1e7f-48da-becf-16f7b420eeec)


## Additional Information
Thank you. 

Fixes #132
